### PR TITLE
Create dummy pages

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,62 +1,63 @@
...
+*.log
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+#Ipython Notebook
+.ipynb_checkpoints
```
Comment (mknecht) [inline .gitignore:63]: @admiral96 please change your editor settings to use linux line endings. ask kerwin about those.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
-# mulungwishi-webhook
-Webhook for the [Mulungwishi Fertilizer Project](https://www.youtube.com/watch?v=qkBsuUHCFWo)
+# mulungwishi-webhook
+Webhook for the [Mulungwishi Fertilizer Project](https://www.youtube.com/watch?v=qkBsuUHCFWo)
```
Comment (mknecht) [inline README.md:2]: same here

File: app/templates/404.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{% extends "base_error.html" %}
+{% block error_content %}
+	<h1>Error 404</h1>
+	<br><br><h2>The page you are looking was not found. Please check your URL again or go <a href="/">back</a>.</h2>
```
Comment (mknecht) [inline app/templates/404.html:4]: the complete verb is “to look for”

File: app/templates/403.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{% extends "base_error.html" %}
+{% block error_content %}
+	<h1>Error 403</h1>
+	<br><br><h2>Page is forbidden. Please go <a href="/webhooks.engagespark.com">back</a> on the previous page.</h2>
```
Comment (mknecht) [inline app/templates/403.html:4]: “go back to”

File: app/views.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+def index():
+    return render_template('index.html')
+
+
+@app.route('/input/<input>')
+def print_user_input(input):
+	query = input.split('=')
+	variable = query[0]
+	value = query[1]
+	return u'Your query is {} which is equal to {}'.format(variable, value)
```
Comment (mknecht) [inline app/views.py:15]: please don't create an extra url for it — the whole idea is that ppl will (accidentally or out of curiosity) visit the _actual_ webhook page — and that one has to show something.

File: app/views.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+from flask import render_template
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/input/<input>')
+def print_user_input(input):
```
Comment (mknecht) [inline app/views.py:11]: please don't shadow a builtin

File: app/views.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+	query = input.split('=')
```
Comment (mknecht) [inline app/views.py:12]: there's a more canonical way to do this — look up tuple unpacking :)

File: config.py [hunk 8]
```diff
@@ -0,0 +1,2 @@
+WTF_CSRF_ENABLED = True
+SECRET_KEY = 'Cthulhuphant_can_cause_annihilation'
```
Comment (mknecht) [inline config.py:2]: that does not make sense, unless I am missing something — the server will not request the page before doing a webhook request :)

File: runner_test.py [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+	def tearDown(self):
+		pass
+
+	def test_valid_url(self):
+		result = self.app.get('/')
+		self.assertEqual(result.status_code, 200)
+
+	def test_invalid_url_server_error(self):
+		result = self.app.get('/input/invalid')
+		assert b'Error 500' in result.data
```
Comment (mknecht) [inline runner_test.py:21]: we should discuss the bytestrings here … we really should just use string

File: runner_test.py [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+		
+	def test_invalid_url_page_not_found(self):
+		result = self.app.get('/a')
+		assert b'Error 404' in result.data
+
+	def test_query_input(self):
+		result = self.app.get('/input/q=query')
+		print(result)
+		self.assertEqual(b'Your query is q which is equal to query', result.data)
```
Comment (mknecht) [inline runner_test.py:30]: … especially here, what happens when you send unicode?

File: runner_test.py [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+		result = self.app.get('/input/invalid')
+		assert b'Error 500' in result.data
+		
+	def test_invalid_url_page_not_found(self):
+		result = self.app.get('/a')
+		assert b'Error 404' in result.data
+
+	def test_query_input(self):
+		result = self.app.get('/input/q=query')
+		print(result)
```
Comment (mknecht) [inline runner_test.py:29]: no prints in PRs, ever

File: runner_test.py [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class LaunchingTest(unittest.TestCase):
+
+	def setUp(self):
+		self.app = app.test_client()
+		self.app.testing = True
+
+	def tearDown(self):
+		pass
```
Comment (mknecht) [inline runner_test.py:13]: why?

**Comment by mknecht** [comment]
too much stuff — and 1 PR instead of 3 — closing! let's revisit this on monday

File: .gitignore (hunk 1 above)
Comment (dashmug) [inline .gitignore:63]: @admiral96 You have to set it on your "Preferences -- User" in SublimeText.

**Pull request closed without merging**